targets/unity-v2/source/ExampleTestProject/Assets/Testing/Tests/Shared/Uunit/UUnitIncrementalTestRunner.cs

```diff
@@ -74,17 +74,20 @@ private void OnSuiteFinish()
         }
 #else
         private void OnSuiteFinish()
+        {
+            if (postResultsToCloudscript)
+                PostTestResultsToCloudScript();
+            else
+                OnCloudScriptSubmit(null);
+        }
+
+        private void FaultRunIfFailed()
```

Comment:
My only remaining comment would be the function-order. It seems that the functions in this file are in sequential order, in terms of the order that they are called in the testing process.
For that reason, it seems FaultRunIfFailed should be moved to the end, after QuitTesting (as it's only called by QuitTesting).